fenix/app/src/main/java/org/mozilla/fenix/translations/TranslationsStore.kt

```diff
@@ -0,0 +1,34 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.translations
+
+import mozilla.components.browser.state.selector.findTab
+import mozilla.components.browser.state.store.BrowserStore
+import mozilla.components.concept.engine.translate.DetectedLanguages
+import mozilla.components.support.base.log.logger.Logger
+
+/**
+ * Manages coordinating the retrieval of data for use in the views.
+ *
+ * @param browserStore The browser store to request information from.
+ * @param tabId The tab id requesting data.
+ */
+class TranslationsStore(
+    private val browserStore: BrowserStore,
+    private val tabId: String,
```

Comment:
If we call methods only from BrowserStore, I don't think a translation store is needed. We can do it with a controller. If a TranslationsStore is need we should extend class Store<S : State, A : Action>.

Reply:
Removed it for the controller pattern! I'll also keep that need to extend in mind if we need to add a `TranslationsStore` later.